package.json

```diff
@@ -4,6 +4,7 @@
   "description": "This file MUST ONLY be modified using scripts/js_dependencies.py",
   "dependencies": {
     "bootstrap": "5.1.0",
+    "bootstrap-select-1.14.0": "beta3",
```

Comment:
Looks like the `js_dependencies` script didn't parse the version correctly. But did we need to use the beta version anyway?

Reply:
The reason for using beta is I had the error `Uncaught TypeError: Cannot read property 'Constructor' of undefined` when using  1.3.18 , which is fixed in 1.14.0-beta3.
https://github.com/snapappointments/bootstrap-select/issues/2439#issuecomment-1271279871
Is it ok for me to manually update "bootstrap-select-1.14.0": "beta3",  to "bootstrap-select": "1.14.0-beta3",  ?

Reply:
I think it will need a fix for `collect_framework_deps` in `js_dependencies.py` otherwise it will break future executions of the script.